samples/storage_csm_powerscale_v240.yaml

```diff
@@ -327,3 +327,84 @@ spec:
             # Default value: "INFO"
             - name: "TOPOLOGY_LOG_LEVEL"
               value: "INFO"           
+
+        - name: otel-collector
```

Comment:
otel-collector is a separate component ?

Reply:
> otel-collector is a separate component ?

Yes, it's an individual requirement according to our tickets.